cat-launcher/src-tauri/src/variants/repository/definition.rs

```diff
@@ -0,0 +1,29 @@
+use crate::variants::GameVariant;
```

Comment:
The name of the file, `definition.rs` is not right. See the file names of other repositories and rename this file accordinlgy.